{"name": "执行登录用户名不存在测试用例", "status": "failed", "statusDetails": {"message": "AssertionError: assert '' == '用户名不存在'\n  - 用户名不存在", "trace": "self = <test_cases.test_001_login.TestLogin object at 0x000001983F4F75C8>\ntest_info = {'expected': '用户名不存在', 'pwd': '123456', 'user': 'ceshiehdej'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"72e437048dc71f4bedab5e5e384b484c\")>\n\n    @pytest.mark.parametrize('test_info', login_data_invalid)\n    @pytest.mark.flaky(reruns=1)\n    @allure.feature('登录模块')\n    @allure.story(\"用户名不存在\")\n    @allure.title(\"执行登录用户名不存在测试用例\")\n    def test_login_invalid_user(self, test_info, get_browser):\n        print(\"------执行登录用户名不存在测试用例------\")\n        driver = get_browser\n        expected = test_info['expected']\n        LoginPage(driver).login(test_info[\"user\"], test_info[\"pwd\"])\n        print(\"------------测试数据---------\")\n        print(\"用户名：\", test_info[\"user\"])\n        print(\"密码：\", test_info[\"pwd\"])\n        invalid_msg = LoginPage(driver).get_invalid_msg()\n        print(\"--------断言结果---------\")\n        print(\"实际结果\", invalid_msg)\n        print(\"预期结果\", expected)\n        try:\n            assert invalid_msg==expected\n        except AssertionError as e:\n>           raise e\n\ntest_cases\\test_001_login.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <test_cases.test_001_login.TestLogin object at 0x000001983F4F75C8>\ntest_info = {'expected': '用户名不存在', 'pwd': '123456', 'user': 'ceshiehdej'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"72e437048dc71f4bedab5e5e384b484c\")>\n\n    @pytest.mark.parametrize('test_info', login_data_invalid)\n    @pytest.mark.flaky(reruns=1)\n    @allure.feature('登录模块')\n    @allure.story(\"用户名不存在\")\n    @allure.title(\"执行登录用户名不存在测试用例\")\n    def test_login_invalid_user(self, test_info, get_browser):\n        print(\"------执行登录用户名不存在测试用例------\")\n        driver = get_browser\n        expected = test_info['expected']\n        LoginPage(driver).login(test_info[\"user\"], test_info[\"pwd\"])\n        print(\"------------测试数据---------\")\n        print(\"用户名：\", test_info[\"user\"])\n        print(\"密码：\", test_info[\"pwd\"])\n        invalid_msg = LoginPage(driver).get_invalid_msg()\n        print(\"--------断言结果---------\")\n        print(\"实际结果\", invalid_msg)\n        print(\"预期结果\", expected)\n        try:\n>           assert invalid_msg==expected\nE           AssertionError: assert '' == '用户名不存在'\nE             - 用户名不存在\n\ntest_cases\\test_001_login.py:79: AssertionError"}, "parameters": [{"name": "test_info", "value": "{'user': 'ceshiehdej', 'pwd': '123456', 'expected': '用户名不存在'}"}], "start": 1616552332097, "stop": 1616552356951, "uuid": "6fdaefab-b64a-4d8b-a073-fb53bb2999e7", "historyId": "075f4daf5ac51fdca2fb54fe12eff52f", "testCaseId": "cd0682f6a139d5392c369302963db253", "fullName": "test_cases.test_001_login.TestLogin#test_login_invalid_user", "labels": [{"name": "story", "value": "用户名不存在"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_001_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-G986QGRS"}, {"name": "thread", "value": "16296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_001_login"}]}